docs/source/abundances.ipynb

```diff
@@ -21,8 +21,9 @@
     "import numpy as np\n",
     "from synthesizer.abundances import (\n",
     "    Abundances,\n",
+    "    Elements,\n",
     "    abundance_scalings,\n",
-    "    solar_abundance_patterns,\n",
+    "    reference_abundance_patterns,\n",
```

Comment:
This is going to break `synthesizer-grids`, no?

Reply:
Although I do prefer the naming convention

Reply:
It's already accounted for in a synthesizer-grids branch.